ultraplot/axes/base.py

```diff
@@ -1583,6 +1591,64 @@ def _artist_fully_clipped(self, artist):
             )
         )
 
+    def _format_inset(
+        self,
+        bounds: tuple[float, float, float, float],
+        parent: "Axes",
+        **kwargs,
+    ) -> "tuple | InsetIndicator":
+
+        if version.parse(str(_version_mpl)) >= version.parse("3.10.0"):
+            return self.__format_inset(bounds, parent, **kwargs)
+        return self.__format_inset_legacy(bounds, parent, **kwargs)
+
+    def __format_inset(
+        self,
+        bounds: tuple[float, float, float, float],
+        parent: "Axes",
+        **kwargs,
+    ) -> "InsetIndicator":
+        # Implementation for matplotlib >= 3.10
+        # NOTE: if the api changes we need to deprecate the old one. At the time of writing the IndicateInset is experimental and may change in the future. This would require us to change potentially the return signature of this function.
+        kwargs.setdefault("label", "_indicate_inset")
+
+        # If we already have a zoom indicator we need to update the properties or add them
+        # Note the first time we enter this function, we create the object. Afterwards the function is accessed again but with different updates
```

Comment:
Let's breakup the long comments across multiple lines.

Reply:
My bad it is my editor that autowraps it at 65 :)